src/pdm/installers/installers.py

```diff
@@ -113,7 +120,17 @@ def _create_symlinks_recursively(source: str, destination: str) -> Iterable[str]
             destination_path = os.path.join(destination_root, f)
             if os.path.exists(destination_path):
                 os.remove(destination_path)
-            os.symlink(source_path, destination_path, False)
+            if any(f.endswith(x) for x in [".py", ".pyi", ".pyx", ".ipynb", ".pth"]):
```

Comment:
```suggestion
            if any(f.endswith(TEXT_FILE_SUFFICES)):
```

And make `TEXT_FILE_SUCCFICES` a combination of these suffices(`endswith` can take a tuple as parameter)